tests/cmd/help.toml

```diff
@@ -1,2 +1,2 @@
 bin.name = "cargo-bisect-rustc"
-args = "--help"
+args = "bisect-rustc --help"
```

Comment:
Please also add a test for the `--help` less variant

Reply:
Oh whoops I misread that, I can add a test that starts a bisect run if it's fine to have the tests download a toolchain

Reply:
Ah no need for downloading, can have a test that fails before that point

Reply:
😆 ah that's a great addition, tho not what I meant. I meant testing that `cargo-bisect-rustc` without `--help` actually prints the same thing as the one with `--help`

Reply:
I... completely misunderstood the PR's main description XD I thought just using `cargo-bisect-rustc` should emit the help message ^^ thanks for clarifying

Reply:
I... completely misunderstood the PR's main description XD I thought just using `cargo-bisect-rustc` should emit the help message ^^ thanks for clarifying